impl/src/main/java/com/okta/jwt/impl/jjwt/TokenVerifierSupport.java

```diff
@@ -81,6 +83,10 @@ Duration getLeeway() {
         return leeway;
     }
 
+    JwtParser getJwtParser() {
```

Comment:
rename this to `parser()` and it's backwards compatible, and you don't need to change the other methods in the `*TokenVerifier`

Reply:
done